drivers/net/ethernet/renesas/rswitch.c

```diff
@@ -3660,7 +3660,7 @@ void rswitch_add_ipv4_forward(struct rswitch_private *priv, u32 src_ip, u32 dst_
 {
 	struct rswitch_device *dev;
 	struct rswitch_ipv4_route *routing_list = NULL;
-	struct l3_ipv4_fwd_param param;
+	struct l3_ipv4_fwd_param param = {0};
```

Comment:
Taking into account my previous comments... Could you please verify that not all fields of this struct are initialized below?

Reply:
This was the actual peace of code that caused bugs. After finding it we decided to initialize all stack structs just in case.